### DISCUSSION TITLE
Username did not change.

### DISCUSSION THREAD

**Discussion by MegaMind-png** [community/community#69890] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi :) I decided to change my name on github, but I ran into a strange problem. The name has changed, but as if not completely :) 
If I open my profile page, I will see my new name as the main one (in bold, large letters at the top), and the old name below it. 
However, all other pages and URL links use my old name. You even have to log into your account under your old name. 
I searched all the forums and old discussions, but nothing helped me :( So I'm asking for help from you, maybe you know what to do :) Thank you
![image](https://github.com/community/community/assets/95245166/ffa4e813-7f17-4213-9858-2cad845974a9)

**Comment by thevindu-w** [comment]
Seems you have only changed the display name of the profile. It does not change the username. You can refer to [docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-user-account-settings/changing-your-github-username](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-user-account-settings/changing-your-github-username#changing-your-username) to change your username.

**Reply by devkhishan** [reply]
Hey !! Happy New Year.  I Got an issue that I have 2 accounts on which I need to transfer the username of account 1 to account 2 but I don't know how to do that.  If I change the username of account 1 to something else, Can I use the username of account 2 on account 1 by update username method on account settings

**Comment by itsdotnet** [comment, marked as answer]
You are changing **Name** not **Username**

To change **Username**:

1. Go to **Settings**

2. Tap **Account** on left side of page.

3. Click **Change Username**

4. Then you can change **Username** after confirmation
